tests/test_build_docs_version.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import pytest
+
+from build_docs import Version
+
+
+def test_equality() -> None:
+    # Arrange
+    version1 = Version(name="3.13", status="stable", branch_or_tag="3.13")
+    version2 = Version(name="3.13", status="stable", branch_or_tag="3.13")
+
+    # Act / Assert
+    assert version1 == version2
+
+
+@pytest.mark.parametrize(
+    ("name", "expected"),
+    [
+        ("3.13", "-rrequirements.txt"),
```

Comment:
Why does this have two `r`?
If it's a typo, why is the test still passing?

Reply:
It's another way of writing "-r requirements.txt" for pip and that's what the production code uses:

https://github.com/python/docsbuild-scripts/blob/1e856ea3a38944e9743c11110d67599c3159cb07/build_docs.py#L196

I do prefer writing with a space as it's much clearer. We could change it in a followup.

Reply:
Make sense, I didn't realize it was a command-line arg.